### PR TITLE
Printing

### LINKED ISSUE #276: Printing
Ability to print what they have selected (filters & needs) and the filtered benefits. Instructions included (ex. 1800 number)

https://github.com/gregnb/react-to-print

### PR DESCRIPTION
**Pull request by smcmurtry**
resolves #276 

A bit rough but it works.

Benefits list component needs to be replaced with something that will display the expanded content of the cards with minimal formatting for printing.

### CODE REVIEW

File: components/BB.js [hunk 1]
```diff
@@ -254,8 +283,15 @@ export class BB extends Component {
       this.props.selectedNeeds
     );
 
+    const url = this.getPrintUrl(
```
Comment (sastels) [inline components/BB.js]: maybe name this printUrl or something, we have a lot of urls in this app.

**Comment by sastels** [comment]
Slightly odd behavior when spitting across two pages. We can punt on that for now if you can’t see a quick fix
9c76d61c-59eb-40dd-8a35-87e3c0043eab

**Comment by sastels** [comment]
Actually when we replace benefit list as you mention above we can ensure that it splits correctly. 👍

**Comment by smcmurtry** [comment]
Haha, you really printed! Yes I think we should create a `PrintableBenefitsList` component.

File: __tests__/pages/print_test.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    expect(await axe(html)).toHaveNoViolations();
+  });
+
+  it("renders BenefitList component", () => {
+    expect(mountedPrint().find("BenefitList").length).toEqual(1);
+  });
+
+  it("parses url correctly", () => {
+    props.url.query["patronType"] = "service-person";
+    props.url.query["serviceType"] = "WSV%20(WWII%20or%20Korea)";
```
Comment (sastels) [inline __tests__/pages/print_test.js]: I think you could just put spaces here instead of `%20`? Otherwise wouldn't our app break? 🤔

**Review by sastels — APPROVED** [review]
This is awesome!

**Comment by sastels** [comment]
(opps sorry for yet another merge conflict)

**Pull request merged**